### PR TITLE
updating grayed cols for ABL

### PR DESCRIPTION
**Pull request by elizabeth-jimenez**

### CODE REVIEW

File: src/sass/_availableBidders.scss [hunk 1]
```diff
@@ -215,12 +221,13 @@
 }
 
 .ab-active {
-  // td(2) is the status column, and td(9) is the comment column
-  td:nth-of-type(2), td:nth-of-type(9) {
-    color: #a9acaf;
-    background-color: #cacbce;
-    a {
+  @each $td-info in $abl-gray-config {
```
Comment (elizabeth-jimenez) [inline src/sass/_availableBidders.scss:224]: @mjoyce91 told you i'd steal it 😈

**Comment by elizabeth-jimenez** [comment]
Also fixes how the last column of ABL for Bureau view gets highlighted

File: src/sass/_variables.scss [hunk 2]
```diff
@@ -226,3 +226,13 @@ $angle-down-black: url('#{$asset-path}assets/img/angle-down-black.svg');
 $button-box-shadow: 5px 10px 20px -4px $color-black-box-shadow;
 $bg-register-with-another-bidder: #dbdbdb;
 $bg-overlay-register-with-another-bidder: #636363;
+
+//**
+//* Available Bidders
+//*
+$abl-actions-td: 11;
+$abl-gray-config: (
```
Comment (SidelineCory24) [inline src/sass/_variables.scss:234]: neat 😎

File: src/Components/AvailableBidder/AvailableBidderRow/AvailableBidderRow.jsx [hunk 3]
```diff
@@ -187,6 +187,8 @@ const AvailableBidderRow = (props) => {
     </Tooltip>) : comments;
 
   const getSections = (isModal = false) => {
+    // when adding/removing columns, make sure to update the
+    // $abl-actions-td and $abl-gray-config variables
```
Comment (SidelineCory24) [inline src/Components/AvailableBidder/AvailableBidderRow/AvailableBidderRow.jsx:191]: me😅

Comment (elizabeth-jimenez) [reply src/Components/AvailableBidder/AvailableBidderRow/AvailableBidderRow.jsx:191]: ALL of us.... so, yes, you too ;p

**Comment by elizabeth-jimenez** [comment]
@SidelineCory24 I choose you to take over this PR - take good care of it 👨‍🍼

**Review by mjoyce91 — APPROVED** [review]

**Pull request merged**